<!doctype html>
<html>
<head>
	<meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
	<meta charset="utf-8">
	
		<title>Build Error for iPad » Community Questions &amp; Answers » Appcelerator Developer Center</title>
	
	<meta name="viewport" content="width=device-width,initial-scale=1">
	<link href="../../related/favicon.png" rel="shortcut icon" type="image/png">
	<link href="//fonts.googleapis.com/css?family=Open+Sans:300,400,500,600,700" rel="stylesheet" type="text/css">
	<link href="//d3ilu1xuwhtfe2.cloudfront.net/e8e0ebd/css/style.css" rel="stylesheet" type="text/css">
	<link href="../../related/qa.css" rel="stylesheet" type="text/css">
	<link href="../../related/hybrid.css" rel="stylesheet" type="text/css">
	
	<link href="../../related/overrides.css" rel="stylesheet" type="text/css">
</head>
<body>

<div class='mast'>    <div class='container'>        <div class='row'>            <div class='col-sm-12'>                <div class='masthead'><h1>Titanium Community Questions & Answer Archive</h1><h2>We felt that 6+ years of knowledge should not die so this is the Titanium Community Questions &amp; Answer Archive</h2></div>            </div>        </div>    </div></div>

<main class="container">
	<div class="row">
		<div class="col-sm-12">
			
		</div>
	</div>
	<div class="row">
		<div class="col-sm-12">
			<article class="question-container">
	<div class="question">
		<header>
			<h1>Build Error for iPad</h1>
		</header>
		<section>
			<p>So i keep getting this build error when i try to test on my ipad</p>
<p><code>[ERROR] Error: Traceback (most recent call last):
  File &amp;quot;&amp;#x2F;Library&amp;#x2F;Application Support&amp;#x2F;Titanium&amp;#x2F;mobilesdk&amp;#x2F;osx&amp;#x2F;1.8.1&amp;#x2F;iphone&amp;#x2F;builder.py&amp;quot;, line 871, in main
    provisioning_profile = read_provisioning_profile(pp,o)
  File &amp;quot;&amp;#x2F;&amp;#x2F;Library&amp;#x2F;Application Support&amp;#x2F;Titanium&amp;#x2F;mobilesdk&amp;#x2F;osx&amp;#x2F;1.8.1&amp;#x2F;iphone&amp;#x2F;builder.py&amp;quot;, line 208, in read_provisioning_profile
    f = open(f,&amp;#39;rb&amp;#39;).read()
IOError: [Errno 2] No such file or directory: u&amp;#39;&amp;#x2F;Library&amp;#x2F;MobileDevice&amp;#x2F;Provisioning Profiles&amp;#x2F;1E32B89E-EF6F-4500-B2E6-FA40BFBB2C50.mobileprovision&amp;#39;</code></p>
<p>any help would be greatly appreciated thanks!</p>

		</section>
		<footer>
			<div class="author">
				&mdash; asked <span title="March 15th 2012, 3:20:43 pm">March 15th 2012</span>
				by <span class='authorname'>Alek Taraskewich</span>
			</div>

			
				<ul class="tags">
					
						<li><span class='tagname'>build</span></li>
					
						<li><span class='tagname'>ios</span></li>
					
						<li><span class='tagname'>ipad</span></li>
					
						<li><span class='tagname'>mobile</span></li>
					
				</ul>
			

			<section>
				<h5>0 Comments</h5>
				
			</section>
		</footer>
		<aside class="vote-box vote-box-answered">
			<div class="score"><span>0</span> Votes</div>
			<div class="answers"><span>2</span> Answers</div>
		</aside>
	</div>

	<hr>

	<h3>2 Answers</h3>

	
		<ul class="answers">
			
				<li class="answer">
				
					<article class="accepted-answer" id="answer-233473">
						<header>
							<h2><span class="icon-bg icon-check-1"></span>Accepted Answer</h2>
						</header>
				
						<section>
							<p>what about read&#x2F;write permissions on your project folder?</p>

						</section>
						<footer>
							<div class="author">
								&mdash; answered <span title="March 15th 2012, 3:44:21 pm">March 15th 2012</span>
								by <span class='authorname'>Sebastian Klaus</span><br>
								<a class="icon-bg icon-link" href="question/133870/build-error-for-ipad#answer-233473" rel="permalink">permalink</a>
							</div>

							<h5>6 Comments</h5>
							
								<ul class="comments">
								
									<li class="comment">
										<p><p>i just checked them and they are fine</p>
</p>
										<div class="author">
											&mdash; commented <span title="March 15th 2012, 3:48:07 pm">March 15th 2012</span>
											by <span class='authorname'>Alek Taraskewich</span>
										</div>
									</li>
								
									<li class="comment">
										<p><p>http:&#x2F;&#x2F;developer.appcelerator.com&#x2F;question&#x2F;65611&#x2F;install-error-traced-to-no-such-file-or-directory-when-trying-to-find-the-developer-profile</p>
</p>
										<div class="author">
											&mdash; commented <span title="March 15th 2012, 4:13:47 pm">March 15th 2012</span>
											by <span class='authorname'>Sebastian Klaus</span>
										</div>
									</li>
								
									<li class="comment">
										<p><p>so i think i figured out what the problem is…i opened up xcode and went to devices and checked the properties of my ipad and it gives me this error </p>
<p>The version of iOS on “iPad” does not match any of the versions of iOS supported for development with this installation of the iOS SDK. Please restore the device to a version of the OS listed below, or update to the latest version of the iOS SDK; which is available here.</p>
<p>is there anyway to revert the ios back easily with out having to reset it to factory defaults?</p>
</p>
										<div class="author">
											&mdash; commented <span title="March 15th 2012, 4:14:25 pm">March 15th 2012</span>
											by <span class='authorname'>Alek Taraskewich</span>
										</div>
									</li>
								
									<li class="comment">
										<p><p>so this all works now, i didnt even have to revert, i believe i just formed some kind of update to the ipad in xcode and that did the trick, thanks for the help</p>
</p>
										<div class="author">
											&mdash; commented <span title="March 15th 2012, 4:27:32 pm">March 15th 2012</span>
											by <span class='authorname'>Alek Taraskewich</span>
										</div>
									</li>
								
									<li class="comment">
										<p><p>i like ;)</p>
</p>
										<div class="author">
											&mdash; commented <span title="March 15th 2012, 4:36:11 pm">March 15th 2012</span>
											by <span class='authorname'>Sebastian Klaus</span>
										</div>
									</li>
								
									<li class="comment">
										<p><p>+1 for that tip!</p>
</p>
										<div class="author">
											&mdash; commented <span title="March 15th 2012, 4:53:43 pm">March 15th 2012</span>
											by <span class='authorname'>Nick Milner</span>
										</div>
									</li>
								
								</ul>
							
						</footer>
						<aside class="vote-box vote-box-answered">
							<div class="score"><span>1</span> Vote</div>
						</aside>
					</article>
				</li>
			
				<li class="answer">
				
					<article id="answer-233472">
				
						<section>
							<p>just delete the build&#x2F;iphone folder and perform a full rebuild</p>

						</section>
						<footer>
							<div class="author">
								&mdash; answered <span title="March 15th 2012, 3:30:29 pm">March 15th 2012</span>
								by <span class='authorname'>Sebastian Klaus</span><br>
								<a class="icon-bg icon-link" href="question/133870/build-error-for-ipad#answer-233472" rel="permalink">permalink</a>
							</div>

							<h5>1 Comment</h5>
							
								<ul class="comments">
								
									<li class="comment">
										<p><p>so i did that and when i run it for the first time it gives me this error </p>
<p><code>Traceback (most recent call last):
  File &amp;quot;&amp;#x2F;Users&amp;#x2F;alektaraskewich&amp;#x2F;Library&amp;#x2F;Application Support&amp;#x2F;Titanium&amp;#x2F;mobilesdk&amp;#x2F;osx&amp;#x2F;1.8.1&amp;#x2F;iphone&amp;#x2F;builder.py&amp;quot;, line 1537, in &amp;lt;module&amp;gt;
    main(sys.argv)
  File &amp;quot;&amp;#x2F;Users&amp;#x2F;alektaraskewich&amp;#x2F;Library&amp;#x2F;Application Support&amp;#x2F;Titanium&amp;#x2F;mobilesdk&amp;#x2F;osx&amp;#x2F;1.8.1&amp;#x2F;iphone&amp;#x2F;builder.py&amp;quot;, line 663, in main
    iphone_creator.create(iphone_dir,True)
  File &amp;quot;&amp;#x2F;Users&amp;#x2F;alektaraskewich&amp;#x2F;Library&amp;#x2F;Application Support&amp;#x2F;Titanium&amp;#x2F;mobilesdk&amp;#x2F;osx&amp;#x2F;1.8.1&amp;#x2F;iphone&amp;#x2F;iphone.py&amp;quot;, line 46, in create
    tiapp = TiAppXML(os.path.join(project_dir,&amp;#39;tiapp.xml&amp;#39;))
  File &amp;quot;&amp;#x2F;Users&amp;#x2F;alektaraskewich&amp;#x2F;Library&amp;#x2F;Application Support&amp;#x2F;Titanium&amp;#x2F;mobilesdk&amp;#x2F;osx&amp;#x2F;1.8.1&amp;#x2F;iphone&amp;#x2F;..&amp;#x2F;tiapp.py&amp;quot;, line 52, in __init__
    data = codecs.open(self.file,&amp;#39;r&amp;#39;,&amp;#39;utf-8&amp;#39;,&amp;#39;replace&amp;#39;)
  File &amp;quot;&amp;#x2F;System&amp;#x2F;Library&amp;#x2F;Frameworks&amp;#x2F;Python.framework&amp;#x2F;Versions&amp;#x2F;2.7&amp;#x2F;lib&amp;#x2F;python2.7&amp;#x2F;codecs.py&amp;quot;, line 881, in open
    file = __builtin__.open(filename, mode, buffering)
IOError: [Errno 2] No such file or directory: u&amp;#39;&amp;#x2F;Users&amp;#x2F;alektaraskewich&amp;#x2F;Documents&amp;#x2F;Titanium Studio Workspace&amp;#x2F;testBarcodeApp3&amp;#x2F;build&amp;#x2F;iphone&amp;#x2F;tiapp.xml&amp;#39;</code></p>
<p>then if i clean the project again and run it i get the same error that was first posted</p>
<p>thanks again</p>
</p>
										<div class="author">
											&mdash; commented <span title="March 15th 2012, 3:41:56 pm">March 15th 2012</span>
											by <span class='authorname'>Alek Taraskewich</span>
										</div>
									</li>
								
								</ul>
							
						</footer>
						<aside class="vote-box">
							<div class="score"><span>0</span> Votes</div>
						</aside>
					</article>
				</li>
			
		</ul>
	
</article>

		</div>
		<div class="col-sm-3"></div>
	</div>
	<div class="row"><div class="signoff">The ownership of individual contributions to this community generated content is retained by the authors of their contributions.<br>All trademarks remain the property of the respective owner.</div></div>
</main>


</body>
</html>
